### PR TITLE
Bg fix route

### PR DESCRIPTION
**Pull request by fab-ryan**
### what does this PR do?
- create new route with map and get Distance from map
### some changes happened as will be viewed in picture below:
Screen Shot 2022-07-07 at 15 18 47
- first of all map will be appeared then `operator` choose `origin` and `destination`
- after all of these click on compass icon to close all required filled will be filled
Screen Shot 2022-07-07 at 15 19 09
- I said the field has been auto value from map pop-up
### How to test this?.

- clone this [PR](https://github.com/atlp-rwanda/Dominators-Phantom-FE/pull/39) or this [repo](https://github.com/atlp-rwanda/Dominators-Phantom-BE)

- then run command `git checkout bg-fix-route`

- also run command for installation new packages 
    - `npm install` 
    - `npm run migrate`
    - `npm run seed`

### CODE REVIEW

File: db.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
+{
```
Comment (tjeaneric) [inline db.json]: Remove dummy JSON files

File: package.json [hunk 2]
```diff
@@ -32,12 +32,15 @@
     "email-validator": "^2.0.4",
     "file-loader": "^6.2.0",
     "formik": "^2.2.9",
-    "babel-loader": "^8.2.4",
```
Comment (tjeaneric) [inline package.json:35]: re-add babel loader

File: src/module/actions/usersAction.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { GET_ALL_USERS, GET_ALL_ERROR } from "..";
```
Comment (tjeaneric) [inline src/module/actions/usersAction.js]: delete this file, It was removed

File: src/module/reducers/usersReducer.js [hunk 4]
```diff
@@ -1,5 +1,4 @@
 import { GET_ALL_ERROR, GET_ALL_USERS } from "..";
```
Comment (tjeaneric) [inline src/module/reducers/usersReducer.js]: delete this file also, It was removed.

**Review by tjeaneric — CHANGES_REQUESTED** [review]

File: src/utils/db.js [hunk 5]
```diff
@@ -1,7 +1,13 @@
 export const backendUrl = process.env.BACKEND_URL;
+
+export const db = process.env.BACKEND_URL;
+import { io } from "socket.io-client";
+// export const db = "http://localhost:4000";
 export const token = localStorage.getItem("token");
+
 export const Authorization = `Bearer ${localStorage.getItem("token")}`;
-export const headers = {
```
Comment (tjeaneric) [inline src/utils/db.js:4]: Please re-add this headers constant, It was being used by many other components

**Review by tjeaneric — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 6]
```diff
@@ -11,18 +11,18 @@ import DriverOperator from "./pages/crud-driver-operator/DriverOperator";
 import Roles from "./pages/roles & permissions/Roles";
 import ResetLink from "./pages/ResetLink";
 import ResetPwd from "./pages/ResetPwd";
-import Dashboard from "./pages/dashboard";
-
 import Assigned from "./pages/AssignDriver/Assigned";
 import UnAssigned from "./pages/AssignDriver/UnAssigned";
+import Dashboard from "./pages/Dashboard";
```
Comment (tjeaneric) [inline src/App.js]: this should be `import Dashboard from "./pages/dashboard";`

File: src/components/admin-header/Header.js [hunk 7]
```diff
@@ -7,16 +7,23 @@ import * as BsIcons from "react-icons/bs";
 import * as IoIcons from "react-icons/io";
 import "./Header.css";
 import { useNavigate } from "react-router-dom";
-
-const Header = () => {
+import { getAllNotifications } from "../../module/actions/NotificationAction";
```
Comment (tjeaneric) [inline src/components/admin-header/Header.js]: this should be `import { getAllNotifications } from "../../module/actions/notificationAction";`

File: src/components/notification/NotificationComponent.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import "./notification.css";
+import { useState, useEffect } from "react";
+import { ImCross } from "react-icons/im";
+import moment from "moment";
+import { updateViewNotification } from "../../module/actions/NotificationAction";
```
Comment (tjeaneric) [inline src/components/notification/NotificationComponent.js]: this should be `import { updateViewNotification } from "../../module/actions/notificationAction";`

File: src/pages/notification/index.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
+import "./notification.css";
+import { useState, useEffect } from "react";
+import Header from "../../components/admin-header/Header";
+import Sidebar from "../../components/admin-sidebar/SideBar";
+import { FaBell } from "react-icons/fa";
+import BreadCrumb from "../../components/BreadCrumb/BreadCrumb";
+import { connect } from "react-redux";
+import NotificationComponent from "../../components/notification/notificationComponent";
```
Comment (tjeaneric) [inline src/pages/notification/index.js]: This should be `import NotificationComponent from "../../components/notification/NotificationComponent";`

**Review by tjeaneric — CHANGES_REQUESTED** [review]

File: src/utils/db.js (hunk 5 above)
Comment (fab-ryan) [reply src/utils/db.js:4]: it is already there

File: src/module/reducers/usersReducer.js (hunk 4 above)
Comment (fab-ryan) [reply src/module/reducers/usersReducer.js]: yes I have removed it

File: src/App.js (hunk 6 above)
Comment (fab-ryan) [reply src/App.js]: Yes I did

**Review by tjeaneric — APPROVED** [review]

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
done

**Review by UmuhireJessie — APPROVED** [review]
Finish up with the view one route functionality

**Review by furebo — APPROVED** [review]

**Pull request merged**